react/features/face-tracking/components/FaceTracker.js

```diff
@@ -0,0 +1,378 @@
+import React, { Component } from 'react';
+
+import face from '../trackingjs/data/face';
+import tracking from '../trackingjs/tracking';
+
+const ANOMALY_RECOGNIZED_DELAY = 6000;
+const FACE_PROMPT_DURATION = 4000;
+
+const FACE_TEXT_FONT_SIZE_RATIO = 0.3;
+
+const TRACKING_INITIAL_SCALE = 4;
+const TRACKING_STEP_SIZE = 2;
+const TRACKING_EDGES_DENSITY = 0.1;
+
+/**
+ * React component for tracking face in a video, and displaying prompt if user's
+ * face is not well-positioned for a while.
+ *
+ * @extends Component
+ */
+class FaceTracker extends Component {
```

Comment:
My current idea is to implement two React components: FaceTracker and FacePrompt. VideoInputPreview calls FaceTracker, whose purpose is to track faces in the video. When appropriate, VideoInputPreview triggers a Redux action to toggle the face tracking in FaceTracker. And when bad position is detected, FaceTracker will call FacePrompt in its render() function.
What do you think of this structure? Is it OK?